api/testrun.go

```diff
@@ -264,6 +264,31 @@ func (c *Client) TestRunAbort(testRunUID string) (bool, string, error) {
 	return response.StatusCode < 400, string(body), nil
 }
 
+// TestRunAbortAll sends a request to abort all running test cases.
+func (c *Client) TestRunAbortAll(organisationUID string) (bool, string, error) {
+	payload := url.Values{}
+	payload.Add("id", organisationUID)
```

Comment:
```suggestion
	payload.Add("organisation_uid", organisationUID)
```

I think we should make it more clear what the `id` is. `TestCaseValidate()` sends an `organisation_uid`, so maybe we can use the same here.